gazebo_ros/include/gazebo_ros/gazebo_ros_api_plugin.h

```diff
@@ -139,14 +143,19 @@ class GazeboRosApiPlugin : public SystemPlugin
   void onModelStatesDisconnect();
 
   /// \brief Function for inserting a URDF into Gazebo from ROS Service Call
-  bool spawnURDFModel(gazebo_msgs::SpawnModel::Request &req,gazebo_msgs::SpawnModel::Response &res);
+  bool spawnURDFModel(gazebo_msgs::SpawnModel::Request &req,
+                      gazebo_msgs::SpawnModel::Response &res);
 
   /// \brief Both SDFs and converted URDFs get sent to this function for further manipulation from a ROS Service call
-  bool spawnSDFModel(gazebo_msgs::SpawnModel::Request &req,gazebo_msgs::SpawnModel::Response &res);
+  bool spawnSDFModel(gazebo_msgs::SpawnModel::Request &req,
+                     gazebo_msgs::SpawnModel::Response &res);
 
   /// \brief delete model given name
   bool deleteModel(gazebo_msgs::DeleteModel::Request &req,gazebo_msgs::DeleteModel::Response &res);
 
+  /// \brief delete a given light by name
+  bool deleteLight(gazebo_msgs::DeleteLight::Request &req,gazebo_msgs::DeleteLight::Response &res);
```

Comment:
I don't, actually I would rather put them, but the line just above mine doesn't have any space between parameters.
I was wondering what the standard is when writing new functions, and it seems like throughout the code this happens many times...